server/dirstore_test.go

```diff
@@ -56,7 +57,7 @@ func require_NoError(t *testing.T, err error) {
 func require_Error(t *testing.T, err error) {
 	t.Helper()
 	if err == nil {
-		t.Errorf("require no error, but got: %v", err)
+		t.Errorf("require error, but got none")
```

Comment:
Why Errorf vs Fatalf? Different Go routine?

Reply:
Agreed. I usually don't use anything but Fatal() because there is no point in continuing the test.

Reply:
not sure, this code was moved from account server and originally dependent on the require library.
probably at the time I saw t.Errorf more often.

Fixed for the entire file